server/json/q029.json

```diff
@@ -0,0 +1,29 @@
+{
+  "type": "q029",
+  "edits": [
+    {
+      "lar": {"loanId": "00001"},
+      "stateCode": "36",
+      "countyCode": "021",
+      "censusTract": "0004.02",
+      "recommendedMSA": "00",
+      "verification": ""
```

Comment:
Cool with me.
In the case where they haven't verified either way should it be: `"verified": null`?